src/logic/calculate.js

```diff
@@ -0,0 +1,23 @@
+const calculate = (dataObject, simbol) => {
```

Comment:
- [ ] You will need the `operate` function in the other file here, so please import it (check the other comments regarding why you will be using it).